machine/fsl_t2080rdb/INSTALL

```diff
@@ -0,0 +1,60 @@
+=========================================
```

Comment:
In general hardware vendors should make a top level "vendor" directory under the machine directory and then put their specific machines in that directory.  I know this is a little confusing because of the pre-existing fsl_p2020rdbpca board that I ported to ONIE several years ago.

In the next few days I will move `machine/fsl_p2020rdbpca` to `machine/fsl/fsl_p2020rdbpca`, then the fsl_t2080rdb can go in that directory also.